### ISSUE TITLE
Don't send custom payloads (aka plugin messages) to different versions

### ISSUE THREAD

**Issue by Earthcomputer** [ViaVersion/ViaVersion#1855]
ViaVersion should block non-vanilla custom payloads travelling between clients and servers on different versions, or (better) provide an API to translate them, and block them if they are left untranslated. Not doing this can lead to clients getting kicked or disconnecting from the server, most notably causing [this issue](https://github.com/FabricMC/fabric/issues/849), but other issues as well, especially if the server writes a BlockPos or ItemStack and the client reads it incorrectly, for example.

**Comment by KennyTV** [comment]
We do have a file to remap channel names at will, would mapping them to null resulting in cancellation be what you imagined? I personally wouldn't like cancelling everything preemptively

**Comment by Earthcomputer** [comment]
Yeah, I was thinking of just cancelling anything untranslated. Otherwise it can and does cause issues. Multiconnect blocks all untranslated custom payloads and provides an API for handling blocked ones; in via's case this would mean providing an API for translating the payload itself (as well as the channel).
